docs/getting-started/getting-started-with-services.rst

```diff
@@ -0,0 +1,86 @@
+Getting Started with Dockstore Services
+=======================================
+
+Tutorial Goals
+--------------
+
+- Get familiarized with the concept of services on Dockstore
+- Learn the basics of creating a service using a ``.dockstore.yml`` file
+- Learn how to register a service and create versions for it
+- Publish your service to Dockstore
+
+.. David will add context here.
+
+Services are meant to be long running processes, usually web services or interactive applications, that can be launched by a user. The entire process for creating a service that can launch successfully is complex and dependent on the service or application you are trying to describe; so it is beyond the scope of this tutorial.
+However, this document will outline what is needed to register, update, and publish a service onto Dockstore. Also, please keep in mind that service functionality is currently in beta mode.
```

Comment:
I would add a note box to the top (see https://beta.docs.dockstore.org/en/develop/) that says that the feature is in beta mode.

https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#admonitions